### PR TITLE
[code] Sync'ed device_id table with the table from Linux kernel

### PR DESCRIPTION
**Pull request by dmitryermilov**
Signed-off-by: Dmitry Ermilov <dmitry.ermilov@intel.com>

### CODE REVIEW

File: _studio/shared/src/libmfx_core_vaapi.cpp [hunk 1]
```diff
@@ -63,276 +63,329 @@ typedef struct {
...
     { 0x8A50, MFX_HW_ICL_LP, MFX_GT2 },
-    { 0x8A51, MFX_HW_ICL_LP, MFX_GT2 },
-    { 0x8A52, MFX_HW_ICL_LP, MFX_GT2 },
-    { 0x8A53, MFX_HW_ICL_LP, MFX_GT2 },
-    { 0x8A56, MFX_HW_ICL_LP, MFX_GT1 },
-    { 0x8A57, MFX_HW_ICL_LP, MFX_GT1 },
-    { 0x8A58, MFX_HW_ICL_LP, MFX_GT1 },
+    { 0x8A5C, MFX_HW_ICL_LP, MFX_GT1 },
     { 0x8A59, MFX_HW_ICL_LP, MFX_GT1 },
+    { 0x8A58, MFX_HW_ICL_LP, MFX_GT1 },
```
Comment (dvrogozh) [inline _studio/shared/src/libmfx_core_vaapi.cpp]: what's the sense to reorder them? To me it is much easier when they are ordered

Comment (dmitryermilov) [reply _studio/shared/src/libmfx_core_vaapi.cpp]: > what's the sense to reorder them?

To be able to compare with i915_pciids.h in bcompare

> To me it is much easier when they are ordered

Much easier to do what?

Comment (dvrogozh) [reply _studio/shared/src/libmfx_core_vaapi.cpp]: > Much easier to do what?

To understand what you have added! or dropped.

I had in mind a little different approach to this change. I hoped to just copy kernel header as is and reuse.

Comment (dmitryermilov) [reply _studio/shared/src/libmfx_core_vaapi.cpp]: If you want to review, you'll have to spend some time on it..

Comment (dvrogozh) [reply _studio/shared/src/libmfx_core_vaapi.cpp]: I am spending it already. However, you are heavily changing the code and you don't explain anywhere what you did and how others are supposed to support the code. Again, you reorder device ID list. How someone is supposed to support that? Why someone won't submit PR to reorder them back. You order is unnatural! At least provide README.md and explain what you did and how someone needs to support that/compare/etc.

Comment (dmitryermilov) [reply _studio/shared/src/libmfx_core_vaapi.cpp]: I explained it. Please see above: "To be able to compare with i915_pciids.h in bcompare"
There is a comment in code that "The list should be sync'ed manually with include/drm/i915_pciids.h". You're saying like current order is natural. No, it is not. New one is at least is synced with some reference point.

Comment (dvrogozh) [reply _studio/shared/src/libmfx_core_vaapi.cpp]: > I explained it. Please see above: 
This should be in code base, not in a github comments.

Comment (dmitryermilov) [reply _studio/shared/src/libmfx_core_vaapi.cpp]: NP. Added. Please review,

**Pull request closed without merging**